lib/l10n/app_it.arb

```diff
@@ -0,0 +1,600 @@
+  "collectionSortRating": "Per valutazione",
+
+  "collectionGroupAlbum": "Per album",
+  "collectionGroupMonth": "Per mese",
+  "collectionGroupDay": "Per giorno",
+  "collectionGroupNone": "Non raggruppare",
+
+  "sectionUnknown": "Sconosciuto",
+  "dateToday": "Oggi",
+  "dateYesterday": "Ieri",
+  "dateThisMonth": "Questo mese",
+  "collectionDeleteFailureFeedback": "{count, plural, =1{Impossibile cancellare 1 elemento} other{Impossibile cancellare {count} elementi}}",
+  "collectionCopyFailureFeedback": "{count, plural, =1{Impossibile copiare 1 elemento} other{Impossibile copiare {count} elementi}}",
+  "collectionMoveFailureFeedback": "{count, plural, =1{Impossibile spostare 1 elemento} other{Impossibile spostare {count} elementi}}",
+  "collectionRenameFailureFeedback": "{count, plural, =1{Impossibile rinominare 1 elemento} other{Impossibile rinominare {count} elementi}}",
+  "collectionEditFailureFeedback": "{count, plural, =1{Impossibile modificare 1 elemento} other{Impossibile modificare {count} elementi}}",
+  "collectionExportFailureFeedback": "{count, plural, =1{Impossibile esportare 1 elemento} other{Impossibile esportare {count} elementi}}",
+  "collectionCopySuccessFeedback": "{count, plural, =1{Copiato 1 elemento} other{Copiati {count} elementi}}",
+  "collectionMoveSuccessFeedback": "{count, plural, =1{Spostato 1 elemento} other{Spostati {count} elementi}}",
+  "collectionRenameSuccessFeedback": "{count, plural, =1{Rinominato 1 elemento} other{Rinominati {count} elementi}}",
+  "collectionEditSuccessFeedback": "{count, plural, =1{Modificato 1 elemento} other{Modificati {count} elementi}}",
+
+  "collectionEmptyFavourites": "Nessun preferito",
+  "collectionEmptyVideos": "Nessun video",
+  "collectionEmptyImages": "Nessuna immagine",
+
+  "collectionSelectSectionTooltip": "Seleziona sezione",
+  "collectionDeselectSectionTooltip": "Deseleziona sezione",
+
+  "drawerCollectionAll": "Tutte le collezioni",
+  "drawerCollectionFavourites": "Preferiti",
+  "drawerCollectionImages": "Immagini",
+  "drawerCollectionVideos": "Video",
+  "drawerCollectionAnimated": "Animato",
+  "drawerCollectionMotionPhotos": "Foto in movimento",
+  "drawerCollectionPanoramas": "Panorami",
+  "drawerCollectionRaws": "Foto raw",
+  "drawerCollectionSphericalVideos": "Video a 360°",
+
+  "chipSortDate": "Per data",
+  "chipSortName": "Per nome",
+  "chipSortCount": "Per numero di elementi",
+
+  "albumGroupTier": "Per livello",
+  "albumGroupVolume": "Per volume di archiviazione",
+  "albumGroupNone": "Non raggruppare",
+
+  "albumPickPageTitleCopy": "Copia su album",
+  "albumPickPageTitleExport": "Esportazione su album",
+  "albumPickPageTitleMove": "Sposta su Album",
+  "albumPickPageTitlePick": "Scegli l'album",
+
+  "albumCamera": "Camera",
+  "albumDownload": "Download",
+  "albumScreenshots": "Screenshots",
+  "albumScreenRecordings": "Registrazioni schermo",
+  "albumVideoCaptures": "Video Captures",
```

Comment:
`Video Captures` is the name of the folder where Aves saves frames captured from videos during playback.
Are you sure you want to keep the English?

Reply:
I guess I missed that.. `Riprese video` would be the direct translation, but that wouldn't really let me think to captures taken _during_ a video, rather on a captured video itself.. Maybe something more like `Scatti nei Video` if it isn't too verbose